src/components/EntityInfo/EntityMenu.vue

```diff
@@ -37,12 +37,12 @@
     <!-- View and Change Business Information -->
     <span v-if="!isDisableNonBenCorps && isBusiness && !isHistorical">
       <v-btn
-        id="company-information-button"
+        id="business-information-button"
         small
         text
         color="primary"
-        :disabled="!isAllowed(AllowableActions.BUSINESS_INFORMATION)"
-        @click="promptChangeCompanyInfo()"
+        :disabled="isChangeBusinessInfoDisabled"
+        @click="promptChangeBusinessInfo()"
```

Comment:
This is to enable the button in specific scenarios. The click event handler may then block the action and display a dialog instead.

Ditto for voluntary dissolution below.